src/ServiceControl/Recoverability/Grouping/DomainHandlers/StoreHistoryHandler.cs

```diff
@@ -35,7 +35,7 @@ public void Handle(RetryOperationCompleted message)
                 retryHistory.AddToUnacknowledged(new UnacknowledgedRetryOperation
                 {
                     RequestId = message.RequestId,
-                    RetryType = message.RetryType,
+                    OperationType = message.RetryType,
```

Comment:
If Operation is a generalisation of Retry and Archive, I would expect something called OperationType to consists of either `Retry` or `Archive`, not `Single Message`, `Group`, `All`  or `Multiple messages`.